benchmarking/benchmark.py

```diff
@@ -314,39 +315,39 @@ def run_experiments(config):
             logger.info('running experiment: {}'.format(current_experiment))
             if repetition != 1:
                 logger.info('\trepetition {} out of {}'.format(rep + 1, repetition))
-            result = run_single_experiment(server, config, threshold, sizes, current_experiment)
+            result = run_single_experiment(config, threshold, sizes, current_experiment)
             results['experiments'].append(result)
 
     return results
 
 
-def run_single_experiment(server, config, threshold, sizes, experiment):
+def run_single_experiment(config, threshold, sizes, experiment):
     result = {}
     credentials = {}
     run = {}
     logger.info("Starting time: {}".format(time.asctime()))
     nb_parties = len(sizes)
     try:
-        credentials = rest_client.project_create(server, config['schema'], 'groups',
+        credentials = rest_client.project_create(config['schema'], 'groups',
                                                  "benchy_{}".format(experiment), parties=nb_parties)
         # upload clks
         upload_binary_clks(config, sizes, credentials)
```

Comment:
ideally, that should be done by the rest_client as well.

Reply:
Agree => https://github.com/data61/anonlink-client/issues/23